dsgrid/config/project_config.py

```diff
@@ -86,10 +87,15 @@ class DimensionsModel(DSGBaseModel):
         ),
         default=[],
     )
+    all_in_one_supplemental_dimensions: List[DimensionType] = Field(
+        title="all_in_one_supplemental_dimensions",
+        description="dsgrid will auto-generate dimensions containing all records for these dimensions.",
```

Comment:
"dsgrid will auto-generate dimensions that aggregate across all records for the base dimensions of these types."